lib/httplib/httplib.go

```diff
@@ -82,7 +82,7 @@ func WithCSRFProtection(fn HandlerFunc) httprouter.Handle {
 		err := csrf.VerifyHTTPHeader(r)
 		if err != nil {
 			log.Warningf("unable to validate CSRF token %v", err)
-			trace.WriteError(w, trace.AccessDenied("access denied"))
+			trace.WriteError(w, trace.Unwrap(trace.AccessDenied("access denied")))
```

Comment:
I don't think you need to unwrap here necessarily since this is a new error, not wrapped one.

Reply:
The unwrap is needed because a trace is generated for new errors also.